cdap-kafka/pom.xml

```diff
@@ -46,6 +46,36 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
```

Comment:
Shouldn't shade. This will affect the packaging in distributed mode. We can just copy all modules and dependencies to a `target/libexec` directory so that cdap-standalone can pickup later (see cdap-spark-core pom.xml).